test/ServiceBus.Tests/TestStreamProviders/EHStreamProviderWithCreatedCacheList.cs

```diff
@@ -22,24 +22,26 @@ public AdapterFactory()
 
             protected override IEventHubQueueCacheFactory CreateCacheFactory(EventHubStreamProviderSettings providerSettings)
             {
-                return new CacheFactory(providerSettings, SerializationManager, createdCaches);
+                return new CacheFactoryForTesting(providerSettings, SerializationManager, createdCaches);
             }
 
-            private class CacheFactory : EventHubQueueCacheFactory
+            private class CacheFactoryForTesting : EventHubQueueCacheFactory
             {
                 private readonly List<IEventHubQueueCache> _caches;
 
-                public CacheFactory(EventHubStreamProviderSettings providerSettings,
+                public CacheFactoryForTesting(EventHubStreamProviderSettings providerSettings,
                     SerializationManager serializationManager, List<IEventHubQueueCache> caches)
                     : base(providerSettings, serializationManager)
                 {
                     _caches = caches;
                 }
-
+                private const int defaultMaxAddCount = 2;
```

Comment:
Add this configuration back and decrease the defaultMaxAddCount to 2 to make it pass more consistently in VSO.